### PR TITLE
Feature/fix pylint

### PR DESCRIPTION
**Pull request by agelsomini**
Fix pylint to 2.14.5.  It appears that out of the box pylint has some issues with python 3.10.*.  So easiest thing for now is version lock it.  Once it gets addressed upstream renovatebot will figure it out and we can advance.

Fix some other pylint changes in the behavior of the checks that were causing breakages.

### CODE REVIEW

**Review by LandonBayer — DISMISSED** [review]

File: robotcontainer.py [hunk 1]
```diff
@@ -41,19 +42,20 @@
...
-from commands.shooter.tarmacshot import TarmacShot
 from commands.shooter.stopaimsystem import StopMovingParts
 
 from commands.auto.fivebrstandard import FiveBRStandard
 from commands.auto.fourblnoninvasive import FourBLNoninvasive
 from commands.auto.twoblhangerouttake import TwoBLHangerOuttake
 from commands.auto.threebrstandard import ThreeBRStandard
+from commands.shooter.decreaseshooterspeed import DecreaseShooterSpeed
+from commands.shooter.increaseshooterspeed import IncreaseShooterSpeed
+from commands.shooter.resetshooteroffset import ResetShooterOffset
```
Comment (caloisio) [inline robotcontainer.py:58]: Should these be moved up to line 50?

File: robotcontainer.py [hunk 2]
```diff
@@ -41,19 +42,20 @@
 from commands.shootball import ShootBall
 from commands.defensestate import DefenseState
 
-from commands.indexer.holdball import HoldBall
 from commands.intake.autoballintake import AutoBallIntake
 from commands.intake.deployintake import DeployIntake
 from commands.intake.retractintake import RetractIntake
 from commands.shooter.aimshootertotarget import AimShooterToTarget
 from commands.shooter.aimshootermanual import AimShooterManually
-from commands.shooter.tarmacshot import TarmacShot
```
Comment (caloisio) [inline robotcontainer.py:50]: Was this line supposed to be removed?

Comment (agelsomini) [reply robotcontainer.py:50]: Yes, it was an unused import.  Its possible that older version of pylint missed it.

File: robotcontainer.py (hunk 1 above)
Comment (agelsomini) [reply robotcontainer.py:58]: Yes, let me do that.  Weird that this movement was good enough to silence the error

**Review by caloisio — APPROVED** [review]

**Review by lmaxwell24 — APPROVED** [review]

**Pull request merged**